gwpy/cli/qtransform.py

```diff
@@ -148,6 +149,9 @@ def fformat(x):  # float format
                 return '[{0}]'.format(', '.join(map(fformat, x)))
             if isinstance(x, Quantity):
                 x = x.value
+            elif isinstance(x, str):
```

Comment:
how does `fformat` ever get called with a `str`? if its via `qrange` I think a better switch would be to set `type=float` in the `--qrange` option definition in line 101.

Reply:
This is one of your changes. I needed this to go further.
I'll look into it.

Reply:
thanks @areeda, I should have been a bit clearer, reading back my message it sounds rude, sorry. I'm just surprised that this hasn't been caught before, can you post a command-line that reproduces it?